vector/src/test/scala/spec/geotrellis/vector/io/json/GeoJsonSpec.scala

```diff
@@ -202,9 +202,8 @@ class GeoJsonSpec extends FlatSpec with Matchers {
     val t1 = jsonGeom.extractGeometries[Polygon]
     t1 should be (Seq(poly1))
 
-    // This should not work
```

Comment:
This commit reverts 3d7cd56, since I think we don't need to change behavior in terms of the current PR.